scipy/stats/stats.py

```diff
@@ -5389,6 +5397,258 @@ def _betai(a, b, x):
 
 
 #####################################
+#       STATISTICAL DISTANCES       #
+#####################################
+
+def wasserstein(u_values, v_values, u_weights=None, v_weights=None):
+    r"""
+    Compute the first Wasserstein distance between two one-dimensional
+    distributions.
```

Comment:
would be good to shorten this a bit and make this fit one line. Sphinx extracts that line as summary in the html docs.